### ISSUE TITLE
The functions `roc_auc_score` and `prc_auc_score` don't work as expected

### ISSUE THREAD

**Issue by afard** [vertica/VerticaPy#671]
These two functions throw similar errors, at least in my case. The following example fails for these two functions, but has no problem with other functions for classification metrics like `accuracy_score`.

from verticapy.machine_learning.metrics.classification import  roc_auc_score
from verticapy.datasets import load_iris
from verticapy.learn.ensemble import RandomForestClassifier
iris_vd = load_iris()
model = RandomForestClassifier("multi_m3", max_features="max", max_depth=3, n_estimators=3, sample=1)
model.fit(iris_vd, ["SepalLengthCm", "SepalWidthCm", "PetalLengthCm", "PetalWidthCm"], "Species")
model.predict(vdf=iris_vd, X=["SepalLengthCm", "SepalWidthCm", "PetalLengthCm", "PetalWidthCm"], name="y_score")
roc_auc_score(y_true = "Species",  y_score = "y_score",  input_relation = iris_vd, average = "macro", labels = model.classes_)

**Comment by oualib** [comment]
this error is incorrect as y_score should be the list of the different probabilities.
Here is the corrected example:
```python
from verticapy.machine_learning.metrics.classification import roc_auc_score
from verticapy.datasets import load_iris
from verticapy.learn.ensemble import RandomForestClassifier
iris_vd = load_iris()
model = RandomForestClassifier("multi_m3", max_features="max", max_depth=3, n_estimators=3, sample=1)
model.drop()
model.fit(iris_vd, ["SepalLengthCm", "SepalWidthCm", "PetalLengthCm", "PetalWidthCm"], "Species")
model.predict_proba(vdf=iris_vd, X=["SepalLengthCm", "SepalWidthCm", "PetalLengthCm", "PetalWidthCm"])
roc_auc_score(y_true = "Species", 
              y_score = ['"randomforestclassifier_multi_m3_irissetosa"', '"randomforestclassifier_multi_m3_irisversicolor"', '"randomforestclassifier_multi_m3_irisvirginica"'], 
              input_relation = iris_vd, 
              average = "macro", 
              labels = model.classes_)
```
However, the error message is incorrect, I'll do a PR to change it.
